routes/basic_db_functions_route.py

```diff
@@ -0,0 +1,12 @@
+from fastapi import APIRouter
+from DAL.basic_db_functions import BasicDbFunctions
+
+router = APIRouter()
+
+
+@router.get('/')
```

Comment:
if the point of this endpoint is to store some data in mong, your endpoint name and function name need to be better.

Reply:
As said before, this endpoint will be removed later anyway, so there is no harm from accepting it.